docs/VR/GetLanguage/assets/GetLanguage.txt

```diff
@@ -1,44 +1,26 @@
 participant App
 participant SDL
 participant HMI
-
-HMI->SDL:BC.OnReady
+App->SDL:RegisterAppInterface
+HMI->>SDL:BC.OnReady
 SDL->HMI:VR.IsReady
-HMI-->SDL:VR.IsReady (code:0)
+HMI-->SDL:VR.IsReady (`SUCCESS`)
 note over SDL:<align:center>`HMICapabilitiesCacheFile`\nis not available
 SDL->SDL:Create `HMICapabilitiesCacheFile`
+note over SDL:<align:center>GetLanguage data\nis not available\nin `HMICapabilitiesCacheFile`
 
-opt Use of cached capabilities data
-
-opt On system start up
-note over SDL:<align:center> Capabilities data is not available\nin`HMICapabilitiesCacheFile`
+opt GetLanguage - Happy Path
 SDL->HMI:VR.GetLanguage
-HMI-->SDL:VR.GetCapabilities (params)
-SDL->SDL:Cache received data\nin `HMICapabilitiesCacheFile`
-App->SDL:GetLanguage
-SDL->SDL:Load cached data\nfrom **`HMICapabilitiesCacheFile`**
-SDL-->App:      GetLanguage (params)          
-end
-
-opt In subsequent ignition cycles
-note over SDL:<align:center> Capabilities data is available\nin`HMICapabilitiesCacheFile`
-App->SDL:GetLanguage
-SDL->SDL:Load cached data\nfrom **`HMICapabilitiesCacheFile`**
-SDL-->App:     GetLanguage (params)          
-end
+HMI-->SDL:VR.GetLanguage (`SUCCESS`, params)
+SDL->SDL:<align:center>Cache received data\nin `HMICapabilitiesCacheFile`
+note right of SDL:<align:center>Use received data\nfor App-SDL communication
 end
 
-opt Use of default capabilities data - No response from HMI
-
-note over SDL:<align:center> Capabilities data is not available\nin`HMICapabilitiesCacheFile`
-SDL->HMI:VR.GetLanguage
-note over HMI:<align:center>No response
-App->SDL:GetLanguage
-SDL->SDL:Load default data\nfrom **`hmi_capabilities.json`**
-SDL-->App:      GetLanguage (params)          
-
-note over SDL:<align:center> Requests\ncapabilities data from HMI\nin subsequent ignition cycle
+opt GetLanguage - No response from HMI
+note over HMI:No response
```

Comment:
```suggestion
SDL->HMI:VR.GetLanguage
note over HMI:No response
```

Reply:
@jacobkeeler please see updates in ba730b1